aait/backend/group-2B/Domain/Common/BaseAuditableEntity.cs

```diff
@@ -0,0 +1,7 @@
+namespace SocialSync.Domain.Common;
+
```

Comment:
why do you add this entity? You can add the fields to the BaseEntity.cs.

Reply:
Noted. The file has been removed and consolidated into one.